cockatrice/src/gameselector.cpp

```diff
@@ -26,7 +27,7 @@
 #include <QVBoxLayout>
 
 GameSelector::GameSelector(AbstractClient *_client,
-                           const TabSupervisor *_tabSupervisor,
```

Comment:
I've been reading through and don't fully understand why we removed the const-ness of the tabSupervisor everywhere

Reply:
opening a user context menu has an option to show that user's games in a game selector

Reply:
Since `GameSelector`'s `TabSupervisor`is const can only call const member functions of `TabSupervisor` but couldn't figure out how to make `switchToGameTabIfAlreadyExists` const because it uses the `setCurrentIndex` of `TabSupervisor`'s parent `QTabWidget` to switch to the tab. 

Because `MessageLogWidget`, `ChatView`, and `UserContextMenu` are downstream from `GameSelector` had to also make their `tab_supervisor` member non-constant.